homeassistant/components/edl21/__init__.py

```diff
@@ -1 +1,18 @@
 """The edl21 component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+
+async def async_setup_entry(
```

Comment:
I suggest that you follow the usual pattern, with a PLATFORMS constant:
```suggestion
PLATFORMS = [Platform.SENSOR]

async def async_setup_entry(
```